pkg/provider/aci_test.go

```diff
@@ -1105,3 +1160,106 @@ func TestGetPodWithContainerID(t *testing.T) {
 	assert.Check(t, is.Equal(testsutil.TestImageNginx, pod.Status.ContainerStatuses[0].Image), "Container image in the container status doesn't match")
 	assert.Check(t, is.Equal(util.GetContainerID(&cgID, &testsutil.TestContainerName), pod.Status.ContainerStatuses[0].ContainerID), "Container ID in the container status is not expected")
 }
+
+func TestFilterWindowsServiceAccountSecretVolume (t *testing.T) {
+	cgName := "pod-" + uuid.New().String()
+	cgNamespace := "ns-" + uuid.New().String()
+	mockCtrl := gomock.NewController(t)
+ 	defer mockCtrl.Finish()
+
+	volName:= "fakeVolume"
+	volMountName1:= "fakeVolumeMount1"
+	volMountPath1:= "/mnt/azure"
+	volMountName2:= "fakeVolumeMount2"
+	serviceAccountSecretMountPath:= "/var/run/secrets/kubernetes.io/serviceaccount"
```

Comment:
You can use this [const](https://github.com/virtual-kubelet/azure-aci/blob/master/pkg/provider/aci.go#L44) instead

Reply:
The mount path requires a pointer, so I decided to stick with this